tests/python/pants_test/engine/sources_test_base.py

```diff
@@ -0,0 +1,25 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.base.file_system_project_tree import FileSystemProjectTree
+from pants.engine.fs import PathGlobs, Snapshot, create_fs_rules
+from pants.engine.legacy.graph import eager_fileset_with_spec
+from pants_test.engine.scheduler_test_base import SchedulerTestBase
+
+
+class SourcesTestBase(SchedulerTestBase):
```

Comment:
Rather than bringing along `SchedulerTestBase`, this should maybe require that it be mixed into `TestBase`? We'll probably want to phase out `SchedulerTestBase` over time...

Reply:
Moved to `TestBase`